packages/lakka/lakka_tools/gamecon_gpio_rpi/package.mk

```diff
@@ -0,0 +1,23 @@
+PKG_NAME="gamecon_gpio_rpi"
+PKG_VERSION="26b61caea841dacc1f57587f655288d39f3ffedc"
+PKG_LICENSE="GPL"
+PKG_SITE="https://github.com/h1aji/gamecon_gpio_rpi"
+PKG_URL="${PKG_SITE}.git"
+PKG_DEPENDS_TARGET="toolchain linux"
+PKG_NEED_UNPACK="${LINUX_DEPENDS}"
+PKG_LONGDESC="Gamecon GPIO rpi joystick driver"
+PKG_TOOLCHAIN="manual"
+PKG_IS_KERNEL_PKG="yes"
+
+pre_make_target() {
+  unset LDFLAGS
+}
+
+make_target() {
+  make KDIR=$(kernel_path)
```

Comment:
this should be something like:
```
kernel_make -C $(kernel_path) M=${PKG_BUILD} modules
```

Reply:
```
make_target() {
  kernel_make KDIR=$(kernel_path) \
        -C $(kernel_path) M=${PKG_BUILD} modules
}
```

does that look correct?

Reply:
I think KDIR is not needed, but it does not hurt.